source/system_stm32f4xx.c

```diff
@@ -81,9 +81,15 @@
 #include "stm32f4xx.h"
 #include "hal_tick.h"
 
-#if !defined  (HSE_VALUE) 
-  #define HSE_VALUE    ((uint32_t)8000000) /*!< Default value of the External oscillator in Hz */
-#endif /* HSE_VALUE */
+#ifndef YOTTA_CFG_HARDWARE_EXTERNALCLOCK
+#define YOTTA_CFG_HARDWARE_EXTERNALCLOCK (8000000)
```

Comment:
You'd rather have a compilation error?  I guess that's fair given what happened to me today!